pages/tutor/addnewcourse.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useSession } from 'next-auth/client';
+import { TextField, Button } from '@material-ui/core';
+import {
+    MuiPickersUtilsProvider,
+    KeyboardDatePicker,
+} from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import Layout from '../../components/layout';
+
+type TProps = {};
+
+export default function AddNewCourse(props: TProps) {
+    const [session, loading] = useSession();
+
+    if (typeof window !== 'undefined' && loading) return null;
+    if (!session) {
+        return (
+            <Layout title="Tutor profile">
+                <h1>You must sign in</h1>;
+            </Layout>
+        );
+    }
+    if (session.role === 'student') {
+        return (
+            <Layout title="Tutor profile">
+                <h1>You must be an admin or tutor to see this page</h1>;
+            </Layout>
+        );
+    }
+
+    const [form, setForm] = useState({
+        student: '',
+        status: '',
+        comment: '',
+    });
+
+    const [
+        selectedDateStart,
+        setSelectedDateStart,
+    ] = React.useState<Date | null>(new Date());
```

Comment:
do we really need to use `Date | null`, what if we use just `React.useState<Date>`?